src/utils/functions/convertContactStatus.ts

```diff
@@ -0,0 +1,74 @@
+import { StatusEnum } from 'src/graphql/types.generated';
+
+// Convert a status string into a StatusEnum
+// Statuses may be lowercase and underscored (i.e. "never_contacted") after task phases lands
+// Statuses may be sentence case with spaces (i.e. "Never Contacted") before task phases lands
+export const convertStatus = (
+  status: string | null | undefined,
+): StatusEnum | null => {
+  const foundStatus =
+    Object.values(StatusEnum).find((value) => {
+      return String(value.toLowerCase()) === String(status?.toLowerCase());
+    }) || null;
```

Comment:
Is the `String` coercion or defaulting to `null` necessary? 

```suggestion
  const foundStatus = Object.values(StatusEnum).find(
    (value) => value.toLowerCase() === status?.toLowerCase(),
  );
```